extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -147,7 +143,7 @@ public class SmallRyeGraphQLProcessor {
     private static final List<String> SUPPORTED_WEBSOCKET_SUBPROTOCOLS = List.of(SUBPROTOCOL_GRAPHQL_WS,
             SUBPROTOCOL_GRAPHQL_TRANSPORT_WS);
 
-    private static final int GRAPHQL_WEBSOCKET_HANDLER_ORDER = -10000;
+    private static final int GRAPHQL_WEBSOCKET_HANDLER_ORDER = ManagementInterfaceFilterBuildItem.AUTHORIZATION + 1;
```

Comment:
I don't believe it's the right index. This is for the management interface. We should have a class with the route order constant.

Reply:
Ahh, yes, it looks like I confused myself with all the positives and negatives. (As filters seem to be inverted from positive to negatives when being added to a route).

Is this the constant that you had in mind? This ensures that WebSocket handling occurs following [this step](https://github.com/rubik-cube-man/quarkus/blob/29f26d34957217d964ab8a2f40c3c4f3a66e1aad/extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/HttpSecurityProcessor.java#L277-L278).